src/view-components/view-utils.tsx

```diff
@@ -3,75 +3,54 @@ import React from "react";
 import Dashboard from "./dashboard/dashboard.component";
 import TabbedView from "./tabbed-view/tabbed-view.component";
 
-import { coreWidgets, coreDashboards, coreTabbedViews } from "./core-views";
 import Widget from "./widget/widget.component";
+import {
+  widgetDefinitions as coreWidgetDefinitions,
```

Comment:
We should probably rename widgetDefinitions to coreWidgetDefinitions, to avoid aliasing every time before use.

Reply:
Sounds good